symphony-bdk-core-invokers/symphony-bdk-core-invoker-api/src/main/java/com/symphony/bdk/core/api/invoker/ApiException.java

```diff
@@ -3,83 +3,104 @@
 import lombok.Getter;
 import org.apiguardian.api.API;
 
+import javax.net.ssl.HttpsURLConnection;
 import javax.ws.rs.core.GenericType;
+
 import java.net.HttpURLConnection;
+import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
 
 /**
  * Main exception raised when invoking {@link ApiClient#invokeAPI(String, String, List, Object, Map, Map, Map, String, String, String[], GenericType)}.
- *
+ * <p>
  * Initially generated by the OpenAPI Maven Generator
  */
 @Getter
 @API(status = API.Status.STABLE)
 public class ApiException extends Exception {
 
-    private int code = 0;
-    private Map<String, List<String>> responseHeaders = null;
-    private String responseBody = null;
+  private int code = 0;
```

Comment:
Be careful as @symphony-hong is also changing the name of this module in `//`

Reply:
Space changes removed